src/CalculationParameters.js

```diff
@@ -1,8 +1,9 @@
 import { Madhab } from './Madhab';
 import HighLatitudeRule from './HighLatitudeRule';
+import { PolarNightMidnightSunResolver } from './PolarNightMidnightSunResolver';
 
 export default class CalculationParameters {
-    constructor(methodName, fajrAngle, ishaAngle, ishaInterval, maghribAngle) {
+    constructor(methodName, fajrAngle, ishaAngle, ishaInterval, maghribAngle, polarNightMidnightSunResolver) {
```

Comment:
The constructor is used for creating the pre-defined params of a calculation method. I think we can omit `polarNightMidnightSunResolver` from the initial params as the use case would almost always be as follows:

```
var params = adhan.CalculationMethod.MuslimWorldLeague()
params.polarNightMidnightSunResolver = PolarNightMidnightSunResolver.aqrabBalad
```

Reply:
Let's do that.